lib/rules/indent.js

```diff
@@ -135,7 +135,8 @@ class BinarySearchTree {
      * Creates an empty tree
      */
     constructor() {
-        this._rbTree = createTree();
+        this._rbTree = new OrderedMap();
+        this._rbTreeEnd = this._rbTree.end();
```

Comment:
nits: we could rename it to a better name - it's no longer a rb-tree. :)

Reply:
Good idea. :D